packages/salesforcedx-vscode-core/src/commands/util/parameterGatherers.ts

```diff
@@ -19,6 +19,9 @@ import { SfdxPackageDirectories } from '../../sfdxProject';
 import { workspaceUtils } from '../../util';
 import { RetrieveDescriber } from '../forceSourceRetrieveMetadata';
 
+export const CONTINUE = 'CONTINUE';
+export const CANCEL = 'CANCEL';
```

Comment:
just a couple of constants that get verified in the unit tests so I exported them here.